geniza/corpus/dates.py

```diff
@@ -392,6 +398,9 @@ def standardize_date(historic_date, calendar):
     if match:
         date_info = match.groupdict()
         year = int(date_info["year"])
+        # Seleucid is Anno Mundi - 3449
+        if calendar == Calendar.SELEUCID:
+            year = year + 3449
```

Comment:
According to Marina's comment https://github.com/Princeton-CDH/geniza/issues/886#issuecomment-1252775572, we don't need to do any special handling for those, but I can certainly test it. Should I just input a Feb 29 date into https://www.muqawwim.com/ and check if the Seleucid result converts back to CE correctly?

Reply:
Anno Mundi calendar handles leap years differently - they add a leap month every 19 years.

Reply:
@blms here's the convertdate code for calculating Anno Mundi leap year: https://github.com/fitnr/convertdate/blob/master/src/convertdate/hebrew.py#L63-L74 

The leap month is Adar Bet or Adar II. My concern is that this leap year calculation doesn't match the Seleucid offset.  I couldn't tell from Marina's answer if she understood what I was asking (maybe she was also thinking of Gregorian calendar leap years). I think you'd have to look for an Adar Bet Seleucid date to figure out the Seleucid leap year offset, but maybe Marina or other team members would have ideas for other ways to investigate.

Reply:
Done!

> I wasn't confident about the muqawimm code because it's not clear to me that it promises anything more than year precision for Seleucid (and the logic is the same as what you implemented). All of these different historical date conversion codebases are based on the same source - both muqawimm and convertdate are based on the Fourmilab code by John Walker.

Fascinating, I had no idea.